src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -320,7 +325,9 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
         });
 
         Handlebars.registerHelper('datetime', function(dateTime) {
-            return new Handlebars.SafeString(new Date(dateTime).toLocaleTimeString());
+            // round to the nearest minute
```

Comment:
This was the [most upvoted approach for moment.js rounding](http://stackoverflow.com/a/28561371), although apparently it has a problematic edge case when it should roll over. You'd think this would be made easy. I'll dig some more for another approach.